packages/target-tauri/runtime-tauri/runtime.ts

```diff
@@ -265,42 +292,29 @@ class TauriRuntime implements Runtime {
     this.store = store
     this.currentLogFileLocation = await invoke('get_current_logfile')
 
-    listen<string>('locale_reloaded', event => {
-      this.onChooseLanguage?.(event.payload)
-    })
-
-    listen<string>('showAboutDialog', () => {
-      this.onShowDialog?.('about')
-    })
-    listen<string>('showSettingsDialog', () => {
-      this.onShowDialog?.('settings')
-    })
-    listen<string>('showKeybindingsDialog', () => {
-      this.onShowDialog?.('keybindings')
-    })
-
-    type sendToChatArguments = {
-      options: {
-        text: string | null | undefined
-        file: { fileName: string; fileContent: string } | null
+    events.onmessage = event => {
```

Comment:
@WofWca these are the if statements I mean. also there is another reason for making the dedicated channel, code wise it is easier and cleaner, because the listener for the jsonrpc part is in a different place